.github/quarkus-bot.yml

```diff
@@ -64,42 +68,38 @@ triage:
               && !matches("hibernate.validator", title)
               && !matches("hibernate.search", title)
               && !matches("hibernate.reactive", title)
+              && !matches("reactive", titleBody)
```

Comment:
I'm not sure about this `titleBody` thing. I'm afraid anything using RESTEasy Reactive will be a problem.

Reply:
I know, there will be false negatives. But currently we have so many false positives, it's really frustrating. I get the impression that half of the Hibernate Reactive issues end up being labelled with `area/hibernate-orm`.

The criteria will never be perfect, but I think it's more useful to say that if the issue mentions "reactive", it's more likely to be about Hibernate Reactive than about Hibernate ORM.

Reply:
I updated the PR to address your comment.